views/pages/utilities/colors.blade.php

```diff
@@ -129,6 +129,16 @@
                 Darkest Text
             @endtypography
         </div>
+
+        <div class="o-grid-4@sm o-grid-3@md o-grid-2@lg o-grid-2@xl u-text-align--center">
+            @typography([
+                'element' => 'h6',
+                'variant' => 'h4',
+                'classList' => ['u-color__text--primary-contrasting', 'u-color__bg--primary']
+            ])
+            Primary Contrasting Text
+            @endtypography
+        </div>
```

Comment:
Would there be value to have a background color primary-contrasting? 
This would make it possible to have inverted styling.